src/projections/analysis/Analysis.java

```diff
@@ -154,7 +154,9 @@ public void initAnalysis(String filename, Component rootComponent)
 		  tachyonShifts = new TachyonShifts(getLogDirectory());
 		  
 		  // Build Summary Data
-		  if (hasSumFiles()) {
+      // If rootComponent == null, we are just checking if the file
+      // exists (via doExitAfterFileLoad), and don't bother running the SumAnalyzer
```

Comment:
Spacing for the comments is inconsistent with the surrounding code.

Reply:
fixed.